index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Formula 1 Get Together Conference</title>
+    <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="hamburger-logo">
+    
+            <div class="header-container">
+                <img src="images/mobile_menu1.svg" class="ham">
+                <nav>
+                <img src="images/F1_logo.png" class="logo">
+                </nav>
+            </div>
+         
+
+        <nav class="menu" id="menu-pop">
+            <img src="images/Icon-close.svg" alt="" class="close-button">
+            <ul class="nav-content">
+                <li><a href="about.html">About</a></li>
+                <li><a href="#program" id="nav-grey">Program</a></li>
+                <li><a href="#join" id="nav-grey">Join</a></li>
+                <li><a href="#sponsor" id="nav-grey">Sponsor</a></li>
+                <li><a href="#news" id="nav-grey">News</a></li>
+            </ul>
+        </nav>
+
```

Comment:
[REQUIRED] You should have a top navbar with social links and a bottom navbar with your logo on the left and nav links on the right. You don't have the top part in yours. Kindly fix this so that your design can be closer to the template
![image](https://github.com/MarkYegon7/Project-Capstone/assets/62938735/51958756-3938-4487-83d3-9f2362217147)

Reply:
Introduced the top navbar with social links